WorkloadManagementSystem/Service/JobMonitoringHandler.py

```diff
@@ -495,10 +495,10 @@ def export_getJobParameter(jobID, parName):
   types_getJobParameters = [[int, long, list]]
 
   @staticmethod
-  def export_getJobParameters(jobIDs):
+  def export_getJobParameters(jobIDs, parName=None):
```

Comment:
there is the `getJobParameter` function, which takes a parameter name

Reply:
Yes I know. The thing is that the `getJobParameter` only works with 1 job at a time, and the return value is different.

Honestly, the correct way should be to rename this method in `export_getJobsParameters` but I didn't want here.

Reply:
OK. Can you add the parameter documentation?